editor.planx.uk/src/@planx/components/MultipleFileUpload/Modal.tsx

```diff
@@ -147,7 +148,17 @@ const SelectMultiple = (props: SelectMultipleProps) => {
     >
       <InputLabel
         id={`select-mutliple-file-tags-label-${uploadedFile.id}`}
-        sx={{ top: "15%", color: (theme) => theme.palette.text.primary }}
+        sx={{
+          top: "16%",
+          textDecoration: "underline",
+          color: (theme) => theme.palette.primary.main,
+          "&[data-shrink=true]": {
```

Comment:
@ianjon3s @jessicamcinchak Just for future reference (no changes needed just now!), I think I would have solved this one like this - 

```ts
const StyledInputLabel = styled(InputLabel)(({ theme, shrink }) => ({
    top: "16%",
    textDecoration: "underline",
    color: theme.palette.primary.main,
    ...(shrink && {
      textDecoration: "none",
      color: theme.palette.text.primary,
      top: "0",
      transform: "translate(14px, -5px) scale(0.85)",
    })
  }));
```

And then using it later...

```jsx
<StyledInputLabel
  id={`select-mutliple-file-tags-label-${uploadedFile.id}`}
  >
  What does this file show?
</StyledInputLabel>
```

The benefits of this would be...
 - Type safety - you could do `&[data-shrink="some string"]`, `&[data-shrink>6]` or `&[data-someMadeUpProp=true]` which would be invalid, but in the example above we know that `shrink` is a `Boolean | undefined` and it's a valid prop for an `InputLabel`
 - We could do more JS-ey stuff with the prop rather than a straight comparison, e.g. `someFunctionThatCalculatesSomething(shrink)`

Reply:
Thanks @DafyddLlyr , I'm keen to use the best method for doing this kind of thing, and also to understand how this is working. I'll probably bother you for a quick chat at some point next week to talk this over.

Reply:
Yeah send me a calendar invite for next week and lets make it happen. It's 100% no bother and upskilling is a key part of all our roles 😄 

If you have a set of questions/problems you can prep/send in advance that would be super helpful, even if it's just very rough or basic.